src/mjolnir/adminbuilder.cc

```diff
@@ -422,7 +419,7 @@ void BuildAdminFromPBF(const boost::property_tree::ptree& pt,
     LOG_ERROR("Error: " + std::string(err_msg));
     sqlite3_free(err_msg);
     sqlite3_close(db_handle);
-    return;
+    return false;
```

Comment:
the number of repetitions of this block of code has reached a sinful level. not saying you should fix it, but who ever did this should be flogged!

Reply:
haha yeah same thought. left a todo